src/app.ts

```diff
@@ -1,6 +1,7 @@
 import express from "express";
 import mongoose, { ConnectOptions } from "mongoose";
-import router from "./routers/router";
+import todoRouter from "./routers/todoRouter";
+import authRouter from "./routers/authRouter";
```

Comment:
Lets create index.js file in all folders like routers, controllers, services, helpers, types, etc and import from there